Update cli command in getting started guide and closes #1138 (#1162)
stellar/stellar-docs: docs/build/guides/dapps/soroban-contract-init-template.mdx

---
title: Develop contract with frontend templates
description: Understand, find, and create your own frontend templates for use with Stellar CLI's `stellar contract init` command
---

<head>

>   <title>Develop contract witn frontend templates</title>

Author: Thanks for catching the typo!

(You may wonder why it makes this unpopular choice. A fair question! The team wanted to balance actual utility with broad approachability. Not everyone learning Stellar and Soroban is familiar with React, or any other UI library. It also demonstrates that core Soroban libraries all work with any JavaScript project.)

> But note that the CLI's `contract init` command was made to work with a specific set of templates. To use other templates, we will clone other templates from repository:

Reviewer: ```suggestion
To use other templates, we will clone them from their repositories:
```

Reviewer: This change just removes the `contract init` reference, since the fe templates are independent of init now.

Author: Yes, fixed for diction

(You may wonder why it makes this unpopular choice. A fair question! The team wanted to balance actual utility with broad approachability. Not everyone learning Stellar and Soroban is familiar with React, or any other UI library. It also demonstrates that core Soroban libraries all work with any JavaScript project.)

To use other templates, we will clone them from their repositories, and then copy these files into the root of the existing `soroban-hello-world` directory:

```bash
# the following is NOT a valid git repository, and will not work 😄

> git clone https://whatever.cool/project/you/want

Reviewer 2: It's pretty important not to link out to websites or repos that don't exist or we don't have control or some trust in. Anyone owning this domain could host a malicious project at this path in the hope of capturing folks who happen to copy and paste this code snippet.

The only domain we should link to as an example should be example.org which is maintained by IANA for that purpose.

Author: Let's use the real repo. I added in the latest commit.

That's it! You have an NPM project with an `initialize.js` script (and its supporting cast). That's all you really need!

Let's try it. First, make sure you're running a local Stellar network. Start Docker Desktop or whatever alternative you prefer, then:

```bash
stellar network container start local
```

> Let's include one extra contract. Go to [the increment example](https://github.com/stellar/soroban-examples/tree/main/increment) and copy increment files into the root directory.

Author: This PR doesn't impact `storing-data.mdx`. Unrelated to the current fix of broken getting started guide.
